tests/App/AppKernel.php

```diff
@@ -107,6 +110,17 @@ protected function configureContainer(ContainerBuilder $container, LoaderInterfa
         }
 
         $loader->load(__DIR__.'/config/services.php');
+
+        /*
+         * TODO: Remove all this and related Types when support for Symfony 4.4 is dropped.
```

Comment:
We could drop Sf 4.4 now no ?

Reply:
I would prefer to do it on another PR. Also I would like to do it for all bundles at once.